src/main.tsx

```diff
@@ -1,19 +1,69 @@
-import { StrictMode } from 'react';
-import { createRoot } from 'react-dom/client';
-import { RouterProvider, createHashRouter } from 'react-router-dom';
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import { Link, Outlet, RouterProvider, createBrowserRouter } from "react-router-dom";
+import { ErrorBoundary } from "react-error-boundary";
+import { TimerProvider } from "./context/TimerContext";
+import TimersView from "./views/TimersView";
+import DocumentationView from "./views/DocumentationView";
+import AddTimer from "./views/AddTimer";
+import HistoryView from './views/HistoryView';
+import { ErrorPage } from "./views/ErrorPageView";
 
-import App from './App';
-import './index.css';
 
-const router = createHashRouter([
-    {
-        path: '/',
-        element: <App />,
-    },
+const PageIndex = () => {
+  return (
+    <TimerProvider>
```

Comment:
If there's an error in your TimerProvider component, it will not be handled by your ErrorBoundary